### ISSUE TITLE
[Accessibility] MAS 4.2.1: (eclipse/list '') The Name property of a focusable elements must not be null.

### ISSUE THREAD

**Issue by soumyatippapuram** [microsoft/azure-tools-for-java#4285]
The following accessibility issue needs investigation.

[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier
 
GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_JavaToolingforIntelliJ/Eclipse-Benchmark; #MAS4.2.1; #JavatoolingforIntelliJ/Eclipse; #C+E-H; #Benchmark; #A11ySev2; #A11yAuto; #AI4D;  

**Environment Details:**
Eclipse IDE for Enterprise Java Developers.
Version: 2020-03 (4.15.0)
Build id: 20200313-1211

**Solution Installed:** The Azure Toolkit for Eclipse 3.220, Scala IDE 4.7.x

**OS Details:**
OS: Windows (Version 1903, Build: 18362.175)

**MAS Violated:** MAS 4.2.1
**App:** eclipse

**Element path:** edit'' , dialog'' , list ''

**Issue Details:** The Name property of a focusable element must not be null. [Section 508 502.3.1](https://www.access-board.gov/guidelines-and-standards/communications-and-it/about-the-ict-refresh/final-rule/single-file-version#502-interoperability-assistive-technology)

**How To Fix:** Provide a UI Automation Name property that concisely identifies the element.

This accessibility issue was found using Accessibility Insights for Windows, a tool that helps debug and find accessibility issues earlier. Get more information and download this tool at https://aka.ms/AccessibilityInsights.

**MAS Reference:**
[MAS 4.2.1](https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7BEEE97BBD-9990-48A0-8B03-6EA775581AA8%7D&file=MAS%204.2.1%20%E2%80%93%20Object%20Information.docx&action=default&mobileredirect=true)

![Bug_4285_Java Tooling_Your Subscriptions_AI](https://user-images.githubusercontent.com/41108985/78998343-0c367e80-7b66-11ea-901c-80cac9eff3a5.png)

![Bug_4285_Java Tooling_Create Authentication Files-Your Subscriptions_AI](https://user-images.githubusercontent.com/41108985/79003971-66890c80-7b71-11ea-9e99-00ea966f58c0.png)
![Bug_4285_Java Tooling_Service Principle Creation Status_AI](https://user-images.githubusercontent.com/41108985/79003989-70127480-7b71-11ea-9ce5-b12951dec449.png)
![Bug_4285_Java Tooling_Create Authentication Files_AI](https://user-images.githubusercontent.com/41108985/79003997-74d72880-7b71-11ea-8f31-dd2b27733bc5.png)

**Comment by Flanker32** [comment]
For the table issue, we are using table component provided by swt, which should be an external issue as swt didn't provide method for us to set name for table, this issue could also be reproduced in Eclipse UI, here is the snapshots. You could get here by Eclipse -> Windows -> Preferences -> Run/Debug -> String Substitution
![image](https://user-images.githubusercontent.com/12445236/79952898-b0e18600-84ad-11ea-82ed-c7a4592da29b.png)

![image](https://user-images.githubusercontent.com/12445236/79952921-b76ffd80-84ad-11ea-8839-c09f4a06d889.png)

**Comment by Flanker32** [comment]
For the subtitle component issue, the dialog is extended from class `org.eclipse.jface.dialogs.TitleAreaDialog`, which is provided by Eclipse. Here is the related [issue](https://github.com/microsoft/azure-tools-for-java/issues/3611),  you may find related screen shots [here](https://github.com/microsoft/azure-tools-for-java/issues/3611#issuecomment-551441705)

**Comment by Flanker32** [comment]
For the blank cell, this is by design that we will only show the tenentId in the first row, other columns are designed to be empty.
![image](https://user-images.githubusercontent.com/12445236/79953261-29484700-84ae-11ea-877c-c56aaa6f602b.png)

**Comment by Flanker32** [comment]
@umasarath52 The issue for blank dialog title has been fixed in #4325, you may get the [bits](https://azuredownloads.blob.core.windows.net/eclipse/accessibility/WindowsAzurePlugin4EJ-3.22.0-SNAPSHOT.develop.11968408.04-22-2020.zip) here, please help us verify it, thanks!

**Comment by soumyatippapuram** [comment]
The issue for blank dialog is fixed.

![Bug_4285_Java Tooling_AI_Fixed3](https://user-images.githubusercontent.com/41108985/80565581-737d7b00-8a0e-11ea-99ca-c9f3c4202278.png)
![Bug_4285_Java Tooling_AI_Fixed2](https://user-images.githubusercontent.com/41108985/80565583-77110200-8a0e-11ea-96d6-463984b8f4d1.png)
![Bug_4285_Java Tooling_AI_Fixed1](https://user-images.githubusercontent.com/41108985/80565587-7a0bf280-8a0e-11ea-97f2-ff15935ad97c.png)

**Comment by Alexandria-R** [comment]
Approved to close as non blocking

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749653
